lib/utils/CSVDatabase.cpp

```diff
@@ -31,7 +31,6 @@ CSVDatabase::create(
     const std::string& file_name,
     const MPI_Comm comm)
 {
-    Database::create(file_name, comm);
```

Comment:
Csv doesn't really create or open a file with this function, so I'm okay with removing the printing statement here